src/main/webapp/course.html

```diff
@@ -163,18 +186,41 @@ <h1 class="p-4 text-center">Add Your Review</h1>
                 max="5"
                 type="number"
                 class="form-control mb-5"
-                placeholder="Professor Rating (From 0 = Terrible to 5 = Excellent)"
+                placeholder="Difficulty (From 0 = Very Easy to 5 = Very Difficult)"
+                required
               />
             </div>
+            <div class="valid-tooltip">Looks good!</div>
+            <div class="invalid-tooltip">
+              Please input a valid difficulty score.
+            </div>
+            <div class="col">
+              <p class="lead">What was your grade in the class?</p>
+              <select id="grade" name="grade" class="form-control mb-5">
```

Comment:
Suggestion: "I" for Incomplete is a grade I've seen before. Maybe not in college though...? Unsure

Reply:
Great idea will take care of that!